packages/jest-environment-jsdom/src/index.ts

```diff
@@ -47,6 +47,10 @@ class JSDOMEnvironment implements JestEnvironment {
       throw new Error('JSDOM did not return a Window object');
     }
 
+    // In the `jsdom@16`, ArrayBuffer was not added to Window, ref: https://github.com/jsdom/jsdom/commit/3a4fd6258e6b13e9cf8341ddba60a06b9b5c7b5b
+    // Install ArrayBuffer to Window to fix it. Make sure the test is passed, ref: https://github.com/facebook/jest/pull/7626
+    global.ArrayBuffer = ArrayBuffer;
```

Comment:
This change is wrong, since it causes the newly‑created global to have the `ArrayBuffer` constructor from the outer realm, whereas <https://github.com/jsdom/jsdom/commit/3a4fd6258e6b13e9cf8341ddba60a06b9b5c7b5b> made it so that the inner realm has its own `ArrayBuffer` constructor and prototype chain.

Reply:
@ExE-Boss great! Wanna send a PR?

Reply:
@SimenB <https://github.com/facebook/jest/pull/10885>